### PR TITLE
NEWS - update for 2.0.0rc2

### PR DESCRIPTION
**Pull request by hppritcha**
@jsquyres 

Signed-off-by: Howard Pritchard howardp@lanl.gov

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ompi-release-pr/1245/ for details.

File: NEWS [hunk 1]
```diff
@@ -168,6 +168,15 @@ v2.0.0 final release; see:
...
 - Update man pages for non-blocking sends per MPI 3.1 standard.
   Thanks to Alexander Pozdneev for reporting.
+- Fix problems when building with plfs file system software present.
+- Fix problem when compiling againsts pvfs2.  Thanks to Dave Love for
+  reporting.
+- Fix problems with MPI_Neighbor_alltoall{v,w}.  Thanks to Willem
+  Vermin for reporting this issue.
+- Fix problem with java bindings on OX X 10.11.  Thanks to Alexander
+  Daryin for reporting this issue.
+- Fix various a compilation problems.  Thanks to Marco Atzeri
```
Comment (jsquyres) [inline NEWS]: Should this be "Fix various compilation problems on Cygwin."?

File: NEWS [hunk 2]
```diff
@@ -2921,6 +2930,8 @@ and v1.4.
   from the trunk before v1.0.
 - Fix for --with-wrapper-ldflags handling.  Thanks to Dries Kimpe for
   pointing this out to us.
+- Fix typos in
+- Fix problems building against plfs.
```
Comment (jsquyres) [inline NEWS]: Were these two bullets left in accidentally?

**Comment by hppritcha** [comment]
@jsquyres shouldn't edit on the bus I guess.  Check again.  Thanks.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ompi-release-pr/1247/ for details.

**Pull request merged**